e2e/cypress/integration/channel/message_spec.js

```diff
@@ -104,18 +104,17 @@ describe('Message', () => {
     });
 
     it('M14320 @here., @all. and @channel. (ending in a period) still highlight', () => {
+        // # Login as new user
+        cy.loginAsNewUser().then(() => {
+            // # Create new team and visit its URL
+            cy.apiCreateTeam('test-team', 'Test Team').then((response) => {
+                cy.visit(`/${response.body.name}`);
+            });
+        });
+
         // # Post message
         cy.postMessage('@here. @all. @channel.');
 
-        // * Check that confirm modal is displayed
-        cy.get('#confirmModal').should('be.visible');
```

Comment:
Failing log: Timed out retrying: Expected to find element: '#confirmModal', but never found it. ([link](https://build-push.internal.mattermost.com/job/mattermost-ui-testing/job/mattermost-cypress/97/testReport/M14320%20@here.,%20@all.%20and%20@channel/%20(ending%20in%20a%20period)%20still%20highlight/Message_M14320__here____all__and__channel___ending_in_a_period__still_highlight/)).

Confirmation modal will show up depending on system setting (`EnableConfirmNotificationsToChannel`) and current member's count vs. `NOTIFY_ALL_MEMBERS` constant.

With that, just create new user and use it for this testing since [test case](https://docs.google.com/spreadsheets/d/14qGnjguYJft7vBTzc7MK0Uai50s1NY9xfD9KFC1GDk0/edit#gid=1636776662&range=C3) is particular to message highlighting only.